Commands/DocumentSets/AddDocumentSet.cs

```diff
@@ -15,12 +15,14 @@ namespace SharePointPnP.PowerShell.Commands.DocumentSets
       SortOrder = 1)]
     public class AddDocumentSet : SPOWebCmdlet
     {
-        [Parameter(Mandatory = true)]
+        [Parameter(Mandatory = true, HelpMessage = "The list object where the document set needs to be added")]
         public ListPipeBind List;
 
-        [Parameter(Mandatory = true)] public string Name;
+        [Parameter(Mandatory = true, HelpMessage = "The name of the document set")] 
+        public string Name;
 
-        [Parameter(Mandatory = true)] public ContentTypePipeBind ContentType;
+        [Parameter(Mandatory = true, HelpMessage = "The content type name of the document set")] 
```

Comment:
Better: The name of the content type, its ID or an actual content object referencing to the document set.